helm/vertical-pod-autoscaler-app/Chart.yaml

```diff
@@ -4,14 +4,20 @@ description: A Helm chart for Kubernetes Vertical Pod Autoscaler
 icon: https://s.giantswarm.io/app-icons/k8s-initiator/1/dark.svg
 home: https://github.com/giantswarm/vertical-pod-autoscaler-app
 maintainers:
-- name: sudermanjr
+  - name: giantswarm/team-atlas
+    email: team-atlas@giantswarm.io
 name: vertical-pod-autoscaler-app
 sources:
-- https://github.com/kubernetes/autoscaler
+- https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler
+- https://github.com/cowboysysop/charts/blob/master/charts/vertical-pod-autoscaler/
+dependencies:
+  - name: vertical-pod-autoscaler
+    version: 5.3.0
+    repository: https://cowboysysop.github.io/charts/
 type: application
-version: [[ .Version ]]
+version: 2.5.3
 annotations:
-  repository: https://github.com/FairwindsOps/charts/tree/master/stable/vpa
+  repository: https://github.com/cowboysysop/charts/blob/master/charts/vertical-pod-autoscaler/
```

Comment:
Not sure we need this

Reply:
@mcbenjemaa asked to add this here. Maybe we could let it here